README.md

```diff
@@ -62,6 +62,54 @@ To start the iOS one, run:
 yarn ios
 ```
 
+## Creating a test/beta build
```

Comment:
At some point, it might be nice to add a table of contents to this README with anchors to each section. Not blocking for this PR though.